app/controllers/annotations_controller.rb

```diff
@@ -19,6 +19,13 @@ class AnnotationsController < ApplicationController
   def create
     annotation = @submission.annotations.new(annotation_params)
 
+    # Handling errors
```

Comment:
Don't need this error check since the submission is already stored in the database